Dockerfile

```diff
@@ -5,10 +5,16 @@ WORKDIR /usr/src
 
 RUN apt update && apt install -y vim wget openjdk-17-jdk
 
-COPY requirements.txt .
+RUN git clone https://github.com/NCBI-Codeathons/mlxai-2024-team-saddler.git
+
+#COPY requirements.txt .
 
 RUN pip install --upgrade pip
 RUN pip install wheel setuptools build
-RUN pip install -r requirements.txt
+#RUN pip install -r requirements.txt
+RUN pip install ipython langchain langchain_core langchain-openai molbloom nest_asyncio openai pandas python-dotenv rdkit rmrkl rxn4chemistry streamlit synspace tiktoken
```

Comment:
Do we need `ipython`, `synspace`, and `streamlit`? I do not see them imported anywhere.

Reply:
I kept ipython for now for dev purposes.
For synspace, good point. I will check it out.
Streamlit is needed for the interface that Trey is currently re-writing/adapting from ChemCrow

Reply:
I am unable to check it for now because of the issue with 'SMILES Parse Error'.

Reply:
I'm trying to resolve this issue now.